README.md

```diff
@@ -14,22 +21,339 @@ You may read more about the NEAR EVM in the link above, but suffice it to say it
+
+    near@example.com
+    
+An email has not been sent, but it will appear in the logs of our Terminal tab running NEAR Contract Helper. Navigate to that tab and search for the email address. There will be a line close to that vicinity that says:
+
+>Confirm your activation code to finish creating your account
+
+Below there will appear a message like this:
+
+```
+…
+'1. Confirm your activation code to finish creating your account:\n' +
+    '154005\n' +
+…
+```
+
+Copy the code (it's several digits, in the snippet above it would be `154005`) and use that to answer the prompt in Wallet for the activation code.
+
+The next screen asks to fund an implicit account with at least 3 Ⓝ. Copy the address and
+fund the account with NEAR CLI, replacing the long implicit account below with yours:
+
+    NEAR_ENV=local near send test.near f81247b9492b80284fa2d90dc498153258a6f08e39db6cfa3356abb43a515432 500 --keyPath ~/.near/local/validator_key.json
+
+Expect to see your user "logged in" in the upper right part of the interface. A screen may ask if you'd like to enable two-factor authentication, but we'll leave the contract as-is. 
+
+---
+
+### NEAR Explorer
+
+Prerequisite: [Docker](https://docs.docker.com/get-docker/).
+
+Clone the [NEAR Explorer](https://github.com/near/near-explorer) repository with:
+
+    git clone git@github.com:near/near-explorer.git
+    cd near-explorer
+
+We'll use Docker to run WAMP routing:
+
+    docker-compose build wamp
+    docker-compose up -d wamp
+    
+Then go into the `backend` directory and create a `db` subdirectory.
+
+    cd backend
+    npm install
+    mkdir db
+    
+Start the backend with: 
+    
+    env NEAR_RPC_URL=http://127.0.0.1:3030 WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws WAMP_NEAR_EXPLORER_BACKEND_SECRET=back npm run start
+    
+You can expect to see your screen show logs repeating these type of messages:
+
+```
+…
+Starting regular data stats check from LEGACY_SYNC_BACKEND...
+Regular data stats check from LEGACY_SYNC_BACKEND is completed.
+Starting regular final timestamp check...
+Regular final timestamp check is completed.
+Starting regular node status check...
+Regular node status check is completed.
+…
+```
+
+Go into the `frontend` directory:
+
+    cd ..
+    cd frontend
+    npm install
+    
+Start the frontend on port 3019:
+
+    env WAMP_NEAR_EXPLORER_URL=ws://localhost:8080/ws ./node_modules/.bin/next -p 3019
+
+Then visit the frontend at http://127.0.0.1:3019
+
+---
+
+Lastly, we'll migrate (build and deploy) the Pet Shop to our local node and be able to use the various services we now have running.
+
+Navigate your terminal or command prompt back to the Pet Shop directory.
+
+## Copy key to Pet Shop
+
+Each localnet will have it's own unique key for the account `test.near` which is created. Copy that file into the project following the keystore naming convention:
```

Comment:
"it's" => "its"

Reply:
extra credit points for this, I like to think I'm keen on grammar and missed this one. 🏅 thanks will fix